diff -wur ../../raw/templates/view/index.tpl ./view/index.tpl
--- ../../raw/templates/view/index.tpl	2009-05-20 17:24:52.000000000 +0200
+++ ./view/index.tpl	2009-05-21 12:31:28.000000000 +0200
@@ -2,7 +2,8 @@
 {include file="sidebar.tpl"}
 {include file="columnleftstart.tpl"}
 
-<div class="addicon">
+<div>
+<span class="addicon fr">
   {$createviewform}
   {$createtemplateform}
   <form method="post" action="{$WWWROOT}view/choosetemplate.php">
@@ -13,16 +14,13 @@
     <input type="hidden" name="institution" value="{$institution}">
 {/if}
   </form>
-</div>
-<div class="cl"></div>
-<div>
+</span>
 
 {if $institution}
   {$institutionselector}
 {/if}
-
 {if $views}
-<table id="myviewstable" class="fullwidth">
+<table id="myviewstable">
   <tbody>
 {foreach from=$views item=view}
     <tr class="{cycle values=r0,r1}">
@@ -31,37 +29,31 @@
         <a href="{$WWWROOT}view/delete.php?id={$view.id}" class="fr" id="btn-deletethisview">{str tag="deletethisview" section="view"}</a>
     {/if}
     <h3><a href="{$WWWROOT}view/view.php?id={$view.id}">{$view.title|escape}</a></h3>
-    
-    {if $view.submittedto}
-    <div class="viewitem submitted-viewitem">{$view.submittedto}</div>
-	{else}
 	<div class="viewitem">
+    {if !$view.submittedto}
         <a href="{$WWWROOT}view/edit.php?id={$view.id}" id="editviewdetails">{str tag="editviewnameanddescription" section="view"}</a> 
+    {/if}
 		{if $view.description}
-			<div class="viewitemdesc">{if !$view.submittedto}{/if}
-			{$view.description}</div>
+        {if !$view.submittedto}<br>{/if}
+        {$view.description}
 		{/if}
     </div>
 	<div class="viewitem">
-		<a href="{$WWWROOT}view/blocks.php?id={$view.id}" id="editthisview">{str tag ="editthisview" section="view"}</a>
+        {if !$view.submittedto}<a href="{$WWWROOT}view/blocks.php?id={$view.id}" id="editthisview">{str tag ="editthisview" section="view"}</a><br>{/if}
     	{if $view.artefacts}
-        	<div class="viewitemdesc">{str tag="artefacts" section="view"}:
-        	{foreach from=$view.artefacts item=artefact name=artefacts}<a href="{$WWWROOT}view/artefact.php?artefact={$artefact.id}&amp;view={$view.id}" id="link-artefacts">{$artefact.title|escape}</a>{if !$smarty.foreach.artefacts.last}, {/if}{/foreach}</div>
+        {str tag="artefacts" section="view"}:
+        {foreach from=$view.artefacts item=artefact name=artefacts}<a href="{$WWWROOT}view/artefact.php?artefact={$artefact.id}&amp;view={$view.id}" id="link-artefacts">{$artefact.title|escape}</a>{if !$smarty.foreach.artefacts.last}, {/if}{/foreach}
 		{/if}
 	</div>
-    {/if}
-
     <div class="viewitem">
     <a href="{$WWWROOT}view/access.php?id={$view.id}" id="editviewaccess">{str tag="editviewaccess" section="view"}</a>
     <br>
     {if $view.access}
-       <div class="viewitemdesc"> {$view.access}
+        {$view.access}
         <br>
-	   </div>
     {/if}
-	
     {if $view.accessgroups}
-        <div class="viewitemdesc">{str tag="whocanseethisview" section="view"}:
+        {str tag="whocanseethisview" section="view"}:
         {foreach from=$view.accessgroups item=accessgroup name=artefacts}
         {* this is messy, but is like this so there aren't spaces between links and commas *}
             {if $accessgroup.accesstype == 'loggedin'}
@@ -72,17 +64,18 @@
                     ({$accessgroup.roledisplay}){/if}{elseif $accessgroup.accesstype == 'user'}
                 <a href="{$WWWROOT}user/view.php?id={$accessgroup.id}">{$accessgroup.id|display_name|escape}</a>{/if}{if !$smarty.foreach.artefacts.last},{/if}
         {/foreach}
-        {if $view.template} {str tag=thisviewmaybecopied section=view}{/if}
-		</div>
+        {if $view.template}<br>{str tag=thisviewmaybecopied section=view}{/if}
     {else}
-        <div class="viewitemdesc">
 		{str tag="nobodycanseethisview2" section="view"}
-		</div>
     {/if}
     </div>
-	
+    {if $view.submittedto}
+        <div class="viewitem">
+        {$view.submittedto}
+        </div>
+    {/if}
     {if $view.submitto}
-        <div class="viewitem submit-viewitem">
+        <div class="viewitem">
         {$view.submitto}
         </div>
     {/if}
@@ -92,10 +85,14 @@
   </tbody>
 </table>
 
-{$pagination}
+<div class="center">{$pagination}</div>
 
 {else}
-<div class="message" id="noviews">{if $GROUP}{str tag="noviewstosee" section="group"}{elseif $institution}{str tag="noviews" section="view"}{else}{str tag="youhavenoviews" section="view"}{/if}</div>
+<table id="myviewstable"{if $member} class="groupviews"{/if}>
+  <tr>
+    <td>{if $GROUP}{str tag="noviewstosee" section="group"}{elseif $institution}{str tag="noviews" section="view"}{else}{str tag="youhavenoviews" section="view"}{/if}</td>
+  </tr>
+</table>
 {/if}
 
 </div>
diff -wur ../../raw/templates/view/sharedviews.tpl ./view/sharedviews.tpl
--- ../../raw/templates/view/sharedviews.tpl	2009-05-09 16:55:22.000000000 +0200
+++ ./view/sharedviews.tpl	2009-05-21 12:31:28.000000000 +0200
@@ -29,7 +29,11 @@
     </table>
     <div class="center">{$pagination}</div>
 {else}
-<div class="message">{str tag="noviewstosee" section="group"}</div>
+<table id="myviewstable"{if $member} class="groupviews"{/if}>
+  <tr>
+    <td>{str tag="noviewstosee" section="group"}</td>
+  </tr>
+</table>
 {/if}
 
 
