--- ../../raw/templates/header.tpl	2009-05-09 16:55:22.000000000 +0200
+++ header.tpl	2009-05-21 14:15:20.000000000 +0200
@@ -28,7 +28,6 @@
     <link rel="stylesheet" type="text/css" href="{$cssurl}">
 {/foreach}
     <link rel="stylesheet" type="text/css" href="{theme_path location='style/print.css'}" media="print">
-    <script type="text/javascript" src="{$WWWROOT}js/css.js"></script>
     <link rel="shortcut icon" href="{$WWWROOT}favicon.ico" type="image/vnd.microsoft.icon">
 </head>
 <body>
@@ -63,13 +62,28 @@
     <div id="logo"><a href="{$WWWROOT}"><img src="{theme_path location='images/logo.gif'}" border="0" alt=""></a></div>
     <h1 class="hidden"><a href="{$WWWROOT}">{$hiddenheading|default:"Mahara"|escape}</a></h1>
 {if $MAINNAV}
+	</div>
+    <div id="mainnav-container">
         <div id="mainnav">
         <ul>
 {foreach from=$MAINNAV item=item}
-{if $item.selected}{assign var=MAINNAVSELECTED value=$item}<li class="selected"><a href="{$WWWROOT}{$item.url|escape}">{$item.title|escape}</a></li>{else}<li><a href="{$WWWROOT}{$item.url|escape}">{$item.title|escape}</a></li>{/if}{/foreach}{if $LOGGEDIN}{if $USER->get('admin') || $USER->is_institutional_admin()}{if $ADMIN || $INSTITUTIONALADMIN}<li><a href="{$WWWROOT}">{str tag="returntosite"}</a></li>{elseif $USER->get('admin')}<li><a href="{$WWWROOT}admin/">{str tag="siteadministration"}</a></li>{else}<li><a href="{$WWWROOT}admin/users/search.php">{str tag="useradministration"}</a></li>{/if}{* <li><a href="" onclick="createLoggingPane(); return false;">Create Logging Pane</a></li> *}{/if}<li><a href="{$WWWROOT}?logout">{str tag="logout"}</a></li>{/if}
+{if $item.selected}{assign var=MAINNAVSELECTED value=$item}<li class="selected"><a href="{$WWWROOT}{$item.url|escape}">{$item.title|escape}</a></li>{else}<li><a href="{$WWWROOT}{$item.url|escape}">{$item.title|escape}</a></li>{/if}{/foreach}
+{if $LOGGEDIN}
+{if $USER->get('admin') || $USER->is_institutional_admin()}
+{if $ADMIN || $INSTITUTIONALADMIN}
+<li><a href="{$WWWROOT}">{str tag="returntosite"}</a></li>
+{elseif $USER->get('admin')}
+<li><a href="{$WWWROOT}admin/">{str tag="siteadministration"}</a></li>
+{else}
+<li><a href="{$WWWROOT}admin/users/search.php">{str tag="useradministration"}</a></li>
+{/if}
+{* <li><a href="" onclick="createLoggingPane(); return false;">Create Logging Pane</a></li> *}
+{/if}
+<li><a href="{$WWWROOT}?logout">{str tag="logout"}</a></li>
+{/if}
         </ul>
         </div>
-    </div>
+
     <div id="subnav">
 {if $MAINNAVSELECTED.submenu}
     <ul>
@@ -78,4 +92,5 @@
 {/if}
 {/if}
     </div>
+
     <div id="mainwrapper">
